modules/get-started/pages/licensing/overview.adoc

```diff
@@ -31,27 +31,74 @@ The Enterprise Edition is licensed with the https://github.com/redpanda-data/red
 
 === License keys
 
-Enterprise features require an Enterprise Edition license key, sometimes referred to as Enterprise license, license key, or license. You can evaluate enterprise features with a free 30-day trial. To request a trial license key, to extend your trial period, or to purchase a license, contact https://redpanda.com/try-redpanda?section=enterprise-trial[Redpanda Sales^].
+Enterprise features require an Enterprise Edition license key, sometimes referred to as Enterprise license, license key, or license.
+
+Starting with version 24.3, new Redpanda clusters automatically receive a trial license that's valid for 45 days, allowing unrestricted use of Enterprise features. This evaluation period begins when the cluster is created for the first time. After this period expires, inactive Enterprise features are disabled, and active features enter a restricted (sanctioned) state.
+
+To request a trial license key, to extend your trial period, or to purchase a license, contact https://redpanda.com/try-redpanda?section=enterprise-trial[Redpanda Sales^].
 
 [[self-managed]]
 === Enterprise features in Redpanda Self-Managed
 
-The following enterprise features for Redpanda Self-Managed are activated with a valid Enterprise Edition license key:
+The following table lists the available Enterprise features for Redpanda Self-Managed, along with the sanctions applied if the Enterprise license is invalid or has expired.
+
+[cols="1,2,2"]
+|===
+| Feature | Description | Sanction if License Invalid
```

Comment:
Do we want to use the term "sanction" in our public docs?  cc @mattschumpert

Reply:
Maybe the term "restriction" is a nicer word?

Reply:
I agree. Thanks for the suggestion. Restriction is definitely a better term here.